packages/datadog-plugin-fastify/test/index.spec.js

```diff
@@ -542,7 +542,193 @@ describe('Plugin', () => {
             })
           }
         })
+
+        describe('with tracer config codeOriginForSpansEnabled', () => {
```

Comment:
Is this name Node-specific or has it been standardized? Mostly asking because it exposes implementation details in the name.

Reply:
It's being implemented across the board. It's already implemented in the Java and Python tracer.

Reply:
The environment variable is standardized: `DD_CODE_ORIGIN_FOR_SPANS_ENABLED`

However the concept of initializing the tracer programmatically, and therefore the need to have equivalent names for the config options, I think is something that's unique to the Node.js tracer?

So no, the name `codeOriginForSpansEnabled` is not standardized.

> Mostly asking because it exposes implementation details in the name.

How is `codeOriginForSpansEnabled` exposing an implementation detail?